test_bundle_debug.py

```
@@ -0,0 +1,70 @@
+
+def format_data(data):
+    """Format the processed data."""
+    return f"Formatted: {data}"
+
+
+# ─ Module: mypackage.config ─
+import types
+
+mypackage = types.ModuleType("mypackage")
+mypackage.config = types.ModuleType("mypackage.config")
+exec('"""Configuration module."""\nimport os\nclass Config:\n    """Configuration class."""\n\n    def __init__(self):\n        self.DEBUG = os.environ.get("DEBUG", "false").lower() == "true"\n        self.LOG_LEVEL = os.environ.get("LOG_LEVEL", "INFO")\nconfig = Config()', globals(), mypackage.config.__dict__)
```

Pylint (3.3.7)

[convention] Line too long (312/100)

(C0301)